debugger/server/server_test.go

```diff
@@ -24,11 +24,22 @@ func TestServer(t *testing.T) {
 
 	time.Sleep(10 * time.Millisecond)
 
-	// first open terminal
-	termConn, err := net.Dial("tcp", addr)
+	numRetries := 3
```

Comment:
Nit: Slightly more readable

```suggestion
	const numRetries = 3
```